astrocut/tests/test_cutout_processing.py

```diff
@@ -247,14 +247,18 @@ def test_center_on_path(tmpdir):
     assert primary_header["OBJECT"] == "Test Target"
 
     # Using the default output filename and not giving an image wcs
-    out_file = cutout_processing.center_on_path(path, size, [cutout_file],
-                                                output_path=tmpdir, verbose=False)
+    target = "C/ Targetname" 
```

Comment:
I think it might be nice to either have this as a pytest.mark.parametrize or just a separate test entirely. This test case is already quite long and it calls into `center_on_path` twice but the return data is only tested once at the end, so we aren't actually testing the first case anymore.